DOCS-7855-7856-4.3-Changes-for-SDK (#63)
mulesoft/docs-mule-sdk: modules/ROOT/pages/oauth.adoc

endif::[]
:keywords: mule, sdk, security, oauth

> OAuth 2.0 is the industry standard protocol for authorization. OAuth 2.0 focuses on client developer simplicity while providing specific authorization flows for web applications, desktop applications, mobile phones, and living room devices. For more information on the OAuth 2.0 protocol, see https://oauth.net/2/[https://oauth.net/2/].

Reviewer: ```suggestion
OAuth 2.0 is the industry-standard protocol for authorization. OAuth 2.0 focuses on client developer simplicity while providing specific authorization flows for web applications, desktop applications, mobile phones, and livingroom devices. For more information about the OAuth 2.0 protocol, see https://oauth.net/2/[https://oauth.net/2/].
```

> The SDK provides support for building modules that are OAuth enabled, which means that they establish connections that are authorized through the OAuth protocol. This allows you to hide most of the complexities of the OAuth protocol from the end user and provide a simpler, easier experience that does not force users to go through the learning curve of OAuth.

Reviewer: ```suggestion
The SDK provides support for building modules that are OAuth enabled, which means that they establish connections that are authorized through the OAuth protocol. This enables you to hide most of the complexities of the OAuth protocol from the end user and to spare them the requirement of learning OAuth.
```

OAuth support provided by the SDK is limited to:

* OAuth version 2.
* xref:mule-sdk::authorization-code.adoc[Authorization Code] grant type

> * xref:mule-sdk::client-credentials.adoc[Client Credentials] grant type (Available since version 1.3)

Reviewer: ```suggestion
* xref:mule-sdk::client-credentials.adoc[Client Credentials] grant type (available since version 1.3)
```
